modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -146,6 +154,65 @@
                table-schema->metabase-field-info
                set)})
 
+(def ^:private ^DateTimeFormatter bq-ts-str-formatter
+  "For some crazy reason, the TIMESTAMP String values returned from [[FieldValue]] look like `1577836800.0`, so create
+  a parser for that.  This is basically '<epoch seconds>.<millis>'."
+  (.toFormatter (doto (DateTimeFormatterBuilder.)
```

Comment:
this is really cool. I've not seen this before

Reply:
Yeah, I was also surprised to come across it.  Seems a bit cleaner than string split based parsing (which would theoretically also work fine).